docs/user-guide/microgrid-concepts.md

```diff
@@ -0,0 +1,128 @@
+# Microgrid concepts
+
+A microgrid is a local electrical grid that connects a set of
+electrical components together.  They are often built around a passive
+power consumer, to supplement the electricity consumed from the public
+grid, with on-site power generation or storage systems.
+
+Microgrids can also function in island-mode, without a grid
+connection, or without a local power consumer, but they have to have
+at least one of the two, to be meaningful.
+
+## Frequenz SDK Microgrid Model
+
+The SDK aims to provide an abstract model of the microgrid that allows
+for high-level interactions with microgrid components, without having
+to deal with, or even be aware of location specific details like:
+
+- where the meters are placed,
+- how many batteries,
+- whether there's a grid connection or a passive consumer,
+- what models the inverters are, etc.
+- whether components are having downtimes, because metrics and limits
+  get adjusted automatically when components are having downtimes.
+
+Users of the SDK can develop applications around this interface once
+and deploy anywhere, and the SDK will take care of translating the
+requests and instructions to correspond to the specific microgrid
+configurations.
+
+``` mermaid
+flowchart LR
+
+subgraph Left[Measurements only]
+direction LR
+  grid["Grid Connection"]
+  consumer["Consumer"]
+  pv["PV Arrays"]
+  chp["CHP"]
+end
+
+junction(( ))
+
+subgraph Right[Measurements and control]
+direction LR
+  bat["Batteries"]
+  ev["EV Chargers"]
+end
+
+grid --- junction
+consumer --- junction
+pv --- junction
+chp --- junction
+
+junction --- bat
+junction --- ev
+```
+
+## Grid
+
+This refers to a microgrid's connection to the external Grid.  The
+power flowing through this connection can be streamed through
+[`grid_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.grid_power].
```

Comment:
Completely unrelated, but in this context, now that we have a `microgrid.grid` object, I wonder if we shouldn't publish metrics there, like we do with `battery_pool`. We can do it after 1.0 (just adding also `microgrid.grid.power` and deprecating `logical_meter.grid_power`).

Reply:
true, that makes sense.

Reply:
* #729